lib/internal/Magento/Framework/App/Request/Http.php

```diff
@@ -425,11 +413,8 @@ public function __sleep()
     public function isSafeMethod()
     {
         if ($this->isSafeMethod === null) {
-            if (isset($_SERVER['REQUEST_METHOD']) && (in_array($_SERVER['REQUEST_METHOD'], $this->safeRequestTypes))) {
-                $this->isSafeMethod = true;
-            } else {
-                $this->isSafeMethod = false;
-            }
+            $this->isSafeMethod = isset($_SERVER['REQUEST_METHOD'])
+                && (in_array($_SERVER['REQUEST_METHOD'], $this->safeRequestTypes));
         }
         return $this->isSafeMethod;
```

Comment:
I don't understand the usage of lazy loading there. HTTP request is stateful, so I'm not going to request clearing that.

Reply:
```
if (isset($_SERVER['REQUEST_METHOD']) && (in_array($_SERVER['REQUEST_METHOD'], $this->safeRequestTypes))) {
  $this->isSafeMethod = true;
```

Is it safe to read the code above as `if (true) { $this->isSafeMethod = true; }` ?

and `} else { $this->isSafeMethod = false; }`

We can't just update this to be $this->isSafeMethod = (bool) result of isset($_SERVER['REQUEST_METHOD']) && (in_array($_SERVER['REQUEST_METHOD'], $this->safeRequestTypes)) ?

I appreciate the feedback.

Reply:
Yes, that was actually my point with "Lazy Load" part, as this should immediately return the `bool` value of this `in_array`

Reply:
Ok, I understand now.  Thank you

Reply:
Updated back to previous.